README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/aragon/aragon.svg?branch=master)](https://travis-ci.org/aragon/aragon)
 [![All Contributors](https://img.shields.io/badge/all_contributors-33-orange.svg?style=flat-square)](#contributors)
 
-**🌎🚀 Trusted by over 400 organizations, securing more than $1MM in funds.**
+**🌎🚀 Trusted by over 550 organizations, securing more than $1MM in funds.**
```

Comment:
Kapow 💥 💥 !

Reply:
I would either specify the currency `USD` or preferably refer to an Ethereum native currency like DAI -- "securing more than 1 million DAI in funds."

Reply:
Isn't $1M understandable? Some people don't know about what Dai is

Reply:
Multiple currencies use the `$` symbol so I would specify as to not be so US-centric. And if they do not know what DAI is then they should ;) maybe putting it there will lead to people looking it up and getting educated!

Reply:
Maybe link the word [DAI](https://www.youtube.com/watch?v=CbjBJ_KrknQ) to an introductory video?

Reply:
Sounds good, I don't have any particular preference. I'll merge this one (which is just the number of orgs) but feel free to submit a PR to change the currency as well